node/src/main/scala/coop/rchain/node/grpc.scala

```diff
@@ -22,15 +22,16 @@ import monix.execution.Scheduler
 import com.google.protobuf.ByteString
 import java.io.{Reader, StringReader}
 
-import coop.rchain.node.diagnostics.{JvmMetrics, NodeMetrics}
+import coop.rchain.node.diagnostics.{JvmMetrics, NodeMetrics, StoreMetrics}
 import coop.rchain.rholang.interpreter.errors.InterpreterError
 
 object GrpcServer {
 
   def acquireServer[
       F[_]: Capture: Monad: MultiParentCasper: NodeDiscovery: JvmMetrics: NodeMetrics: Futurable](
       port: Int,
-      runtime: Runtime)(implicit scheduler: Scheduler): F[Server] =
+      runtime: Runtime)(implicit scheduler: Scheduler): F[Server] = {
+    implicit val storeMetricsCapture: StoreMetrics[F] = diagnostics.storeMetrics[F](runtime.store)
```

Comment:
Can you move this to **node.scala** and add `StoreMetrics` to `F[_]]` capabilieties?

Reply:
Sure, updated.